doc/scripts/buildGitbook.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
-# This should be run from the root of the project: ./doc/scripts/buildGitbook.sh
```

Comment:
useful for building the gitbook stuff, can remove the java doc stuff though

Reply:
Put it back in.